torch_nets/models.py

```diff
@@ -101,23 +172,11 @@ def endow_with_netcdf_params(model: Module, nc_file: str):
     """
     data_set = nc.Dataset(nc_file)  # pylint: disable=no-member
 
-    for name, layer in model.named_children():
-        if not isinstance(layer, Linear):
-            continue
-
-        layer_num = int("".join(filter(lambda x: x.isdigit(), name)))
-
-        weight = as_tensor(
-            data_set.variables[f"w{layer_num}"][:],
-            dtype=layer.weight.dtype,
-            device=layer.weight.device,
-        )
-
-        bias = as_tensor(
-            data_set.variables[f"b{layer_num}"][:],
-            dtype=layer.bias.dtype,
-            device=layer.bias.device,
-        )
+    for i, layer in enumerate(l for l in model.modules() if isinstance(l, nn.Linear)):
```

Comment:
This is a nice way to make the code more concise and, IMO, more readable.

Reply:
Do you mean I need to make any change here?

Reply:
No, just saying it is good work :)